comid/tdx-profile/example_pce_refval_test.go

```diff
@@ -221,28 +241,44 @@ func Example_decode_PCE_CBOR() {
 		panic(err)
 	}
 
-	// Output:
+	// output:
```

Comment:
Curiosity: why lowercasing “output”?

https://go.dev/blog/examples has it capitalised.

Reply:
corrected now: thanks